newIDE/app/src/BehaviorsEditor/NewBehaviorDialog.js

```diff
@@ -89,6 +104,15 @@ export default function NewBehaviorDialog({
   const eventsFunctionsExtensionsState = React.useContext(
     EventsFunctionsExtensionsContext
   );
+  const authenticatedUser = React.useContext(AuthenticatedUserContext);
+
+  const installDisplayedExtension = ACHIEVEMENT_FEATURE_FLAG
```

Comment:
I think we can do without the feature flag as it's now added and in production :)

Reply:
True, I'm removing them